.travis.yml

```diff
@@ -7,11 +7,14 @@ cache:
     - $HOME/.composer/cache
 
 php:
-  - 5.6
   - 7.0
   - 7.1
   - hhvm
 
+matrix:
+    allow_failures:
+        - php: hhvm
```

Comment:
Apologies, was bringing it in line with the other repos, will read up on how to force a version of HHVM as don't use it much myself at present.